### ISSUE TITLE
Checkbox inputs missing a hover state (DAC usability feedback)

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2250]
DAC have suggested we add a hover state to the checkbox input. Currently only the label has a cursor change on hover, the input/checkbox itself does not. 

A simple fix would be to change cursor to pointer, but DAC also suggest a thicker border on hover.

Page 76-77 of the June 2022 DAC DS report...

---

## Additional Indicator (Usability)
An additional indicator for elements that can be actioned is not available.

### Reference:
Usability & GOV.UK Design System requirement
Issue ID: DAC_Usability_01
URL: https://ons-design-system.netlify.app/components/accordion/examples/accordion-simple/
Page title: Example
Journey: Journey 1, Step 4, a

Filter checkboxes may be problematic for low vision users determining if the element can be activated. There is no mouse indicator that the element is clickable, such as a cursor change. Changing of cursor or colour on hover helps low vision users know that there is an action that can be committed.

### Current code ref(s):
#data 
```
<input type="checkbox" id="data" class="ons-checkbox__input ons-js-checkbox" value="data">
```

### Low vision comments: 
“When viewing this page, I noticed the check boxes do not have a highlight when I hover the pointer over them. This affects me because of my eye condition, I rely on a highlighted element to help me determine that each check box is ready to be clicked on when I hover the pointer over them. It would benefit me that each check box has a highlighted element surrounding the check box or have an underline to appear/disappear on the text next to the check box when I hover the pointer over them.”

### Solution:
By adding an onhover visual cursor change or making the border of the checkboxes thicker, it can provide additional visual information that the element can be clicked. This will help low vision users navigating the filters available.
